File: app/src/main/java/hr/nas2skupa/eleventhhour/ui/SubcategoryViewHolder.java
package hr.nas2skupa.eleventhhour.ui;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import hr.nas2skupa.eleventhhour.R;
import hr.nas2skupa.eleventhhour.model.Subcategory;

/**
 * Created by nas2skupa on 18/09/16.
 */
public class SubcategoryViewHolder extends RecyclerView.ViewHolder {
    private TextView titleView;


    public SubcategoryViewHolder(View itemView) {
        super(itemView);

        titleView = (TextView) itemView.findViewById(R.id.txt_subcategory_name);
    }

    public void bindToSubcategory(Subcategory subcategory) {
        titleView.setText(subcategory.getName());
    }
}

File: app/src/main/java/hr/nas2skupa/eleventhhour/ui/viewholders/SubcategoryViewHolder.kt
package hr.nas2skupa.eleventhhour.ui.viewholders

import android.support.v7.widget.RecyclerView
import android.view.View
import hr.nas2skupa.eleventhhour.common.model.Subcategory
import kotlinx.android.synthetic.main.item_subcategory.view.*

/**
 * Created by nas2skupa on 18/09/16.
 */
class SubcategoryViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    fun bind(subcategory: Subcategory) = with(itemView) {
        txt_subcategory_name.text = subcategory.localName
    }
}
